internal/entity/common.go

```diff
@@ -150,6 +150,26 @@ type Cursor struct {
 	Limit     int
 }
 
+func NewSeverityFromRating(rating SeverityValues) Severity {
```

Comment:
Please add a comment where this values come from. There might also be an adjustment depending on the CVSS version.
https://nvd.nist.gov/vuln-metrics/cvss

Reply:
I added a comment about the CVSS specification with links, but I'm not sure how I would handle doing adjustments based on which CVSS specification since there is no information being passed other than the rating. Should I just do something else here instead?

Reply:
For now that's okay